### PR TITLE
Fixed: Make tld generic Website cleanse

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
Title

#### Todos
- [ ] Tests
- [] Wiki Updates

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: src/NzbDrone.Core/Parser/Parser.cs [hunk 1]
```diff
@@ -462,7 +462,7 @@ public static class Parser
                                                                 string.Empty,
                                                                 RegexOptions.IgnoreCase | RegexOptions.Compiled);
 
-        private static readonly RegexReplace WebsitePrefixRegex = new RegexReplace(@"^\[\s*[-a-z]+(\.[a-z]+)+\s*\][- ]*|^www\.[a-z]+\.(?:com|net|org)[ -]*",
+        private static readonly RegexReplace WebsitePrefixRegex = new RegexReplace(@"^\[\s*[-a-z]+(\.[a-z]+)+\s*\][- ]*|^www\.[a-z]+(\.[a-z]+)[ -]*",
```
Comment (markus101) [inline src/NzbDrone.Core/Parser/Parser.cs:465]: Are there actually single character TLDs?

**Review by markus101 — CHANGES_REQUESTED** [review]
> A few sentences describing the overall goals of the pull request's commits.

Mmmhmm.

This needs tests.

File: src/NzbDrone.Core/Parser/Parser.cs (hunk 1 above)
Comment (Qstick) [reply src/NzbDrone.Core/Parser/Parser.cs:465]: RFC-952 allows it, but there currently doesn't seem to be any and there is some debate in recent RFC proposals.. Also, they can be alpha numeric but can't be all numeric, and must be under 24 chars. 

```
A "name" (Net, Host, Gateway, or Domain name) is a text string up
to 24 characters drawn from the alphabet (A-Z), digits (0-9), minus
sign (-), and period (.).
```

```
There is an additional rule that essentially requires
     that top-level domain names not be all-numeric.
```

REF current list: https://data.iana.org/TLD/tlds-alpha-by-domain.txt

Maybe `^\[\s*[-a-z]+(\.[a-z]+)+\s*\][- ]*|^www\.[a-z]+(\.(?![0-9]*$)[a-z0-9]{2,24})[ -]*`

**Comment by markus101** [comment]
@bakerboy448 are you going to address the regex?

**Comment by markus101** [comment]
This regex had several failing tests still, I've redone in e273f16c3905e0c2451f43cf98b9b7ad1cbdc777

**Pull request closed without merging**